File: 1-two-sum/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
      unordered_map <int,int> nummap;
      for(int i=0;i<nums.size();i++){
        int sub=target-nums[i];
        if(nummap.count(sub)){
            return {nummap[sub],i};
        }
        nummap[nums[i]]=i;
      }
      
    return {}; }
    };

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ar={}
        for i,val in enumerate(nums):
           rm=target-nums[i]
           if(rm in ar):
            return[i,ar[rm]]
           else:
            ar[val]=i